extensions/sovity-messenger-demo/src/main/java/de/sovity/edc/extension/sovitymessenger/demo/message/Common.java

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ */
+
+package de.sovity.edc.extension.sovitymessenger.demo.message;
+
+public interface Common {
+    String ROOT = "de.sovity.demo.calculator.";
+}
```

Comment:
for a demo it might be better to omit the common prefix to better understand what "names" the messages got